rns_shell/compositor/layers/Layer.h

```diff
@@ -45,7 +45,14 @@ struct PaintContext {
 
 class Layer {
 public:
-
+	
+    float opacity{1.0};
+    float shadowOpacity{1.0};
+    float shadowRadius{0};
+    int  backfaceVisibility;
+    SkSize shadowOffset;
+    SkColor shadowColor;
+    SkM44 transform;
```

Comment:
Need to make it private and way to use directly in RskComponent using friend class. also group the variable accordingly

Reply:
As discuss this changes not required.